aks-kubesphere-linux/azuredeploy.json

```diff
@@ -0,0 +1,506 @@
+            }
+        },
+        "AKSSubnetAddressRange": {
+            "type": "string",
+            "defaultValue": "10.10.128.0/20",
+            "metadata": {
+                "description": "The address range of AKS subnet"
+            }
+        },
+        "AKSServiceAddressRange": {
+            "type": "string",
+            "defaultValue": "10.0.0.0/16",
+            "metadata": {
+                "description": "The address range of AKS service, it must not overlap with any subnet IP ranges"
+            }
+        },
+        "AKSdnsServiceIP": {
+            "type": "string",
+            "defaultValue": "10.0.0.10",
+            "metadata": {
+                "description": "The dns  service IP in AKS"
+            }
+        },
+        "dockerBridgeCidr": {
+            "type": "string",
+            "defaultValue": "172.17.0.1/16",
+            "metadata": {
+                "description": "The bridge cidr for docker。"
+            }
+        },
+        "ClientSubnetName": {
+            "type": "string",
+            "defaultValue": "KubeSphereRG-vnet-client",
+            "metadata": {
+                "description": "The name of new or existing client subnet"
+            }
+        },
+        "ClientSubnetAddressRange": {
+            "type": "string",
+            "defaultValue": "10.10.10.0/24",
+            "metadata": {
+                "description": "The name of new or existing client subnet"
+            }
+        },
+        "AKSresourceName": {
+            "type": "string",
+            "defaultValue": "AKS-KubeSphere",
+            "metadata": {
+                "description": "The name of AKS resource"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_F8s_v2",
+            "metadata": {
+                "description": "virtual machine resource size，default Standard_F8s_v2 - 8vcpu、16GiB"
+            }
+        },
+        "osDiskSizeGB": {
+            "type": "int",
+            "defaultValue": 0,
+            "metadata": {
+                "description": "Disk size (in GB) to provision for each of the agent pool nodes. This value ranges from 0 to 1023. Specifying 0 will apply the default disk size for that agentVMSize."
+            },
+            "minValue": 0,
+            "maxValue": 1023
+        },
+        "dnsPrefix": {
+            "type": "string",
+            "defaultValue": "AKS-KubeSphere-dns",
+            "metadata": {
+                "description": "Optional DNS prefix to use with hosted Kubernetes API server FQDN."
+            }
+        },
+        "kubernetesVersion": {
+            "type": "string",
+            "allowedValues": [
+                "1.17.13",
+                "1.17.16",
+                "1.18.10",
+                "1.18.14",
+                "1.19.6",
+                "1.19.7"
+            ],
+            "defaultValue": "1.18.14",
```

Comment:
use the latest as the defaultValue - these deprecate pretty quickly

Reply:
Thanks for your reminder, I will modify it.